templates/javascript.js.twig

```diff
@@ -57,14 +57,20 @@ CKEDITOR.on('dialogDefinition', function(ev) {
     }
 });
 
+var decodeHTML = function (html) {
```

Comment:
IMO we can use `const` and `let` here :

```
const decodeHTML = function (html) {
     let txt = document.createElement('textarea');
     txt.innerHTML = html;

     return txt.value;
 };
```

Reply:
I would like to be consistent with the rest of the file. WDYT ?

Reply:
Ok, we can modify that in another PR :+1: